brave/brave-core
# Pin to taskbar on Windows

fix https://github.com/brave/brave-browser/issues/24054

* Pin to taskbar from first run or default browser dialog when user want to do it.
* Pin to taskbar from settings.

First run dialog:
![2022-09-22_12h58_10](https://user-images.githubusercontent.com/6786187/191655369-2378438a-aa81-448d-a01b-4d46f2316a34.png)
Default browser prompt:
![2022-09-22_12h56_47](https://user-images.githubusercontent.com/6786187/191655421-9b48fe5f-120f-473a-826e-e02e695043a0.png)
![image](https://user-images.githubusercontent.com/6786187/191906559-e03e69a1-4935-4936-9999-b79da79a7ef8.png)
Settings when not pinned:
![2022-09-22_23h09_34](https://user-images.githubusercontent.com/6786187/191772534-db98d636-b18b-4d10-9e85-e7f9020c0ba2.png)
Settings already pinned:
![image](https://user-images.githubusercontent.com/6786187/191898422-724a2708-9d3f-4a31-aee5-dfb2e2b96e42.png)



Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1.  Launch Brave with clean profile
2. Click `Set Default` button after checking `pin to taskbar`
3. Check shortcut is pinned to taskbar
4. Load brave://settings/getStarted and check settings shows as `pinned`
5. Unpin from taskbar and reload brave://settings/getStarted and click `Pin` button
6. Check pinned to taskbar
7. Unpin from taskbar and set another non-brave browser as a default browser
8. Launch Brave with clean profile
9. Repeat launch till default browser dialog is launched
10. Click `Set Default` button after checking `pin to taskbar`
11. Check shortcut is pinned
